src/utils.js

```diff
@@ -0,0 +1,8 @@
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+
+export const getUserById = userId => usersFromServer
+  .find(({ id }) => id === userId) ?? null;
```

Comment:
There is no need to return `null` here, the `find` method will return `undefined` if it finds nothing.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find